local_planner/src/local_planner.cpp

```diff
@@ -461,9 +450,9 @@ void LocalPlanner::getStateAndTwistInput() {
     ref_body_plan_(i, 2) = z_des_ + ref_ground_height_(i);
 
     // Only adaptive pitch
-    ref_body_plan_(i, 4) = local_footstep_planner_->getTerrainSlope(
-        ref_body_plan_(i, 0), ref_body_plan_(i, 1), ref_body_plan_(i, 6),
-        ref_body_plan_(i, 7));
+    // ref_body_plan_(i, 4) = local_footstep_planner_->getTerrainSlope(
```

Comment:
remove?

Reply:
I want to keep them. There are two ways to align with terrain, here is another way.